### ISSUE TITLE
Is it possible to use github action to build and publish to release?

### ISSUE THREAD

**Issue by linusxiong** [folbricht/routedns#189]
Is it possible to use github action to build and publish to release? i find it has some github action to check and test, so why add some code to enable the workflow to build go and public mutipal platform version to release!

**Comment by folbricht** [comment]
That would be nice, thank you

**Comment by folbricht** [comment]
Yes, this exists now. `.github/workflows/release.yaml` runs goreleaser on any `v*` tag and publishes binaries for linux, darwin, windows and freebsd across amd64, arm64 and arm, plus deb and rpm packages. There are also binary-only armv6, mips and mipsle builds for routers and small devices.

Ordinary pushes to master are built and tested but not released.

Closing as done.
